third_party/net-istio.yaml

```diff
@@ -113,12 +113,48 @@ spec:
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: config.webhook.istio.networking.internal.knative.dev
+  labels:
+    serving.knative.dev/release: "v20200309-53bb214"
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    service:
+      name: istio-webhook
+      namespace: knative-serving
+  failurePolicy: Fail
+  sideEffects: None
+  name: config.webhook.istio.networking.internal.knative.dev
+  namespaceSelector:
+    matchExpressions:
+    - key: serving.knative.dev/release
+      operator: Exists
```

Comment:
cc @trshafer

Reply:
Interesting thanks. I think we will want to have uniform selectors in the webhooks. 
The validating webhook is looking for knative/release and the mutating webhook is looking for knative/configuration.

https://github.com/knative/net-istio/blob/53bb214aeddf3773bf6e838f4016080c44ed811d/config/500-webhook.yaml#L32

https://github.com/knative/net-istio/blob/6bc827c83d937aa8789d8cb394c59918809086f8/config/500-webhook-configmap.yaml#L33

Reply:
I was just giving you a heads up that this was coming into serving :)

The selectors are necessarily different because the objects they act on are different.